confignode/src/main/java/org/apache/iotdb/confignode/service/ConfigNodeCommandLine.java

```diff
@@ -97,18 +95,17 @@ protected int run(String[] args) {
 
   private void doRemoveNode(String[] args) throws IOException {
     LOGGER.info("Starting to remove {}...", ConfigNodeConstant.GLOBAL_NAME);
-    if (args.length != 3) {
-      LOGGER.info("Usage: -r <internal_address>:<internal_port>");
+    if (args.length != 2) {
+      LOGGER.info("Usage: <Node-id>");
```

Comment:
```suggestion
      LOGGER.info("Usage: -r <Node-id>");
```

Reply:
I deem the '-r' somehow redundant, so I removed it in the command